.github/workflows/scheduled-ci.yml

```diff
@@ -0,0 +1,124 @@
+name: Scheduled CI
+
+on:
+  schedule:
+    - cron: '0 0/4 * * *'
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+permissions: {}
+
+defaults:
+  run:
+    shell: bash
+
+jobs:
+  test_browsers:
```

Comment:
All these jobs here lack the "snapshots" keyword. I think this is confusing and we should fix this